source/common/http/conn_manager_impl.cc

```diff
@@ -1766,6 +1766,11 @@ void ConnectionManagerImpl::ActiveStream::recreateStream(
   // Make sure to not check for deferred close as we'll be immediately creating a new stream.
   connection_manager_.doEndStream(*this, /*check_for_deferred_close*/ false);
 
+  auto oldAccount = response_encoder->getStream().account();
```

Comment:
can we follow style guide: https://google.github.io/styleguide/cppguide.html#Variable_Names for naming local variable